flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TestingTaskExecutorGateway.java

```diff
@@ -181,7 +183,7 @@ public void disconnectResourceManager(Exception cause) {
 	}
 
 	@Override
-	public CompletableFuture<TransientBlobKey> requestFileUpload(FileType fileType, Time timeout) {
+	public CompletableFuture<TransientBlobKey> requestFileUpload(FileType fileType, String fileName, Time timeout) {
```

Comment:
If `fileType` is `LOG` or `STDOUT`, it seems that the information in `fileName` won't be relevant. Is that right?